app/videobridge/server/methods/jitsiGenerateToken.js

```diff
@@ -0,0 +1,58 @@
+import { Meteor } from 'meteor/meteor';
+import { settings } from '../../../settings';
+import { jws } from 'jsrsasign';
+
+Meteor.methods({
+	'jitsi:generateAccessToken': (jitsiRoom) => {
```

Comment:
I think to make this truely secure... We should instead pass the room here.  Then inside verify the user is actually allowed to access the room.  Otherwise if I was clever I could gain access to someone's room by simply calling this meteor method and passing the jitsi room id.

we should make sure that only a member of the room the call is started from can access that room.